packages/cfd/src/Containers/trade-modal.tsx

```diff
@@ -0,0 +1,176 @@
+import React from 'react';
+import { Text, Icon, Money } from '@deriv/components';
+import { TTradingPlatformAccounts, TCFDDashboardContainer, TCFDsPlatformType } from 'Components/props.types';
+import { DetailsOfEachMT5Loginid } from '@deriv/api-types';
+import { CFD_PLATFORMS, getCFDAccountKey, isMobile, capitalizeFirstLetter } from '@deriv/shared';
+import { localize, Localize } from '@deriv/translations';
+import { getPlatformQRCode, PlatformsDesktopDownload } from '../Helpers/config';
+import { getTitle, platformsText, mobileDownloadLink } from '../Helpers/constants';
+import SpecBox from 'Components/specbox';
+import PasswordBox from 'Components/passwordbox';
+
+type TTradeModalProps = {
+    mt5_trade_account: Required<DetailsOfEachMT5Loginid>;
+    is_eu_user: boolean;
+    onPasswordManager: (
+        arg1: string | undefined,
+        arg2: string,
+        arg3: string,
+        arg4: string,
+        arg5: string | undefined
+    ) => void;
+    toggleModal: () => void;
+    dxtrade_tokens: TCFDDashboardContainer['dxtrade_tokens'];
+    is_demo: string;
+    platform: TCFDsPlatformType;
+};
+
+const PlatformIconsAndDescriptions = (platform: TCFDsPlatformType, is_demo: string) => {
+    return (
+        <React.Fragment>
+            <Icon
+                icon={platform === 'derivez' ? 'IcBrandDerivEz' : `IcBrand${capitalizeFirstLetter(platform)}`}
+                size={24}
+            />
+            <div className='cfd-trade-modal__desc'>
+                <Text size='xs' line_height='l' className='cfd-trade-modal__desc-heading'>
+                    <Localize
+                        i18n_default_text='Deriv {{platform}} {{is_demo}}'
+                        values={{
+                            platform: platformsText(platform),
+                            is_demo: is_demo ? 'Demo' : '',
+                        }}
+                    />
+                </Text>
+            </div>
+        </React.Fragment>
+    );
+};
+
+const TradeModal = ({
+    mt5_trade_account,
+    is_eu_user,
+    onPasswordManager,
+    toggleModal,
+    dxtrade_tokens,
+    is_demo,
+    platform,
+}: TTradeModalProps) => {
+    const CTraderAndDerivEZDescription = () => {
+        const description = platform === 'derivez' ? 'Deriv EZ' : 'cTrader';
```

Comment:
change this variable name to the more descriptive one
```suggestion
        const platform_name = platform === 'derivez' ? 'Deriv EZ' : 'cTrader';
```